components/datetime/src/fields/length.rs

```diff
@@ -54,6 +56,8 @@ pub enum FieldLength {
     Six,
     /// A fixed size format for numeric-only fields that is at most 127 digits.
     Fixed(u8),
```

Comment:
Question: are we using `Fixed` anywhere? Why do we have it?

Reply:
yeah i have no idea what this is

Reply:
It seems to be something people can supply via skeletons